cql3/selection/selection.cc

```diff
@@ -344,6 +349,7 @@ class selection_with_processing : public selection {
         const selection_with_processing& _sel;
         std::vector<raw_value> _temporaries;
         bool _requires_thread;
+        size_t _input_row_count;
```

Comment:
size_t is logically wrong here (and in the function too). size_t is for "countable things in memory", and this counts things on disk. A theoretical 32-bit scylladb would fail if we'd page in more than 2^32 rows.

In practice it works, but it still sends readers the wrong signal. Should be uint64_t.